.gitignore

```
@@ -4,3 +4,6 @@
 terraform.tfstate*
 terraform.tfvars
 values.dev.yaml
+bin/
+pkg/
+.idea/
```

Should we include control-plane bin and pkg folders too? I think the reason we ignore them is temporary stuff gets created in there in makefile scripts or when we do builds, so might make sense to ignore them inside the module as well
```suggestion
.idea/
control-plane/bin
control-plane/pkg
```

---

maybe it automatically includes control-plane/bin and control-plane/pkg and we don't have to do this? I already see those not being added by git

---

once cli comes in then it'll probably have those dirs too so probably good to have it at the top-level